client/src/layers/UILayers/UILayer.ts

```diff
@@ -0,0 +1,46 @@
+import { createApp } from '@vue/composition-api';
+import { MediaController } from 'vue-media-annotator/components/annotators/mediaControllerType';
```

Comment:
This should be a relative import.  In the published lib, this alias doesn't exist.